src/clarity-angular/wizard/wizard-page.ts

```diff
@@ -119,6 +119,25 @@ export class WizardPage implements OnInit {
     @Output("clrWizardPagePreventDefaultCancelChange") stopCancelChange: EventEmitter <boolean> =
         new EventEmitter();
 
+    // overrides next from the page level, so you can use an alternate function for
+    // validation or data-munging with clrWizardPageOnNext
+    private _stopNext = false;
+    public get stopNext(): boolean {
+        return this._stopNext;
+    }
+    @Input("clrWizardPagePreventDefaultNext")
+    public set stopNext(val: boolean) {
+        let valBool = !!val;
```

Comment:
Do we need `let valBool = !!val;` if `val` is a already boolean parameter as `!!val` will have the same value as the `val` parameter, right?